Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Location/AddEditDrawingsModal.tsx

```diff
@@ -0,0 +1,121 @@
+//******************************************************************************************************
+//  LocationDrawings.tsx - Gbtc
+//
+//  Copyright © 2024, Grid Protection Alliance.  All Rights Reserved.
+//
+//  Licensed to the Grid Protection Alliance (GPA) under one or more contributor license agreements. See
+//  the NOTICE file distributed with this work for additional information regarding copyright ownership.
+//  The GPA licenses this file to you under the MIT License (MIT), the "License"; you may not use this
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  11/06/2024 - Collins Self
+//       Generated original version of source code.
+//
+//******************************************************************************************************
+import { SystemCenter } from "@gpa-gemstone/application-typings";
+import { Input, Select } from "@gpa-gemstone/react-forms";
+import { Modal } from "@gpa-gemstone/react-interactive";
+import React from "react";
+
+interface IProps {
+    Record: SystemCenter.Types.LocationDrawing,
+    Setter: (record) => void,
+    HandleSave: () => void,
+    Show: boolean,
+    SetShow: (show: boolean) => void,
+}
+
+const AddEditDrawingsModal = (props: IProps) => {
+    const [category, setCategory] = React.useState<Array<SystemCenter.Types.ValueListItem>>([]);
+
+    function valid(field: keyof (SystemCenter.Types.LocationDrawing)): boolean {
+        if (field == 'Name')
+            return props.Record.Name != null && props.Record.Name.length > 0 && props.Record.Name.length <= 200;
+        else if (field == 'Link')
+            return props.Record.Link != null && props.Record.Link.length > 0;
```

Comment:
Do these links have a character limit? Check the DB for this.

Reply:
If I'm understanding correctly, no? They are set to `varchar(max)`